### PR TITLE
added a couple of points about EFS

### PR DESCRIPTION
**Pull request by 0xmohit**
Additionally, fixed the incorrect link to "short list" of APIs and
SDKs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -835,9 +835,11 @@ EFS
 ### EFS Basics
 
 -	🐥**EFS** is Amazon’s new (general release 2016) network filesystem.
+-	It is designed to be highly available and durable and each EFS file system object is redundantly stored across multiple Availability Zones.
```
Comment (ThanosBaskous) [inline README.md]: Consistency in capitalization: "Availability Zones" -> "availability zones"

**Comment by 0xmohit** [comment]
@ThanosBaskous fixed.

**Comment by ThanosBaskous** [comment]
LGTM. Thanks @0xmohit! Assigning to @jlevy for second stage review since EFS doesn't have a topic editor.

**Comment by jlevy** [comment]
👍 LGTM as well. Thanks @0xmohit !

**Pull request merged**